project_template/tasks.py

```diff
@@ -163,6 +163,27 @@ class TaskOwnedIncomeFromPensionsTable(CountTableRowsTask):
     child_class = TaskOwnedIncomeFromPensionsRowEntry
 
 
+class TaskOwnedGoodsOrServicesPerOwnerRowEntry(DigitalizationTask):
+    task_form = forms.TranscribeOwnedGoodsOrServicesPerOwnerSingleRowTable
+    template_name = "task/owned_goods_or_services_per_owner.html"
+
+    def save_verified_data(self, verified_data):
+        owned_goods_or_services_per_owner, created = models.OwnedGoodsOrServicesPerOwnerTableEntry.object.gen_or_create(
```

Comment:
The attribute of the Model is "objects", not "object"
Please check the documentation: https://docs.djangoproject.com/en/2.1/topics/db/models/#model-attributes